koku/masu/test/processor/parquet/test_parquet_report_processor.py

```diff
@@ -271,15 +281,11 @@ def test_convert_to_parquet(self, mock_remove, mock_exists):
             self.assertEqual(file_name, "")
             self.assertTrue(data_frame.empty)
 
-        with patch("masu.processor.parquet.parquet_report_processor.get_path_prefix", return_value=""):
-            with patch(
-                "masu.processor.parquet.parquet_report_processor.ParquetReportProcessor.report_type", return_value=None
-            ):
-                with patch(
-                    "masu.processor.parquet.parquet_report_processor.ParquetReportProcessor.prepare_parquet_s3"
-                ):
-                    with self.assertRaises(ParquetReportProcessorError):
-                        self.report_processor_ocp.convert_to_parquet()
+        with patch("masu.processor.parquet.parquet_report_processor.get_path_prefix", return_value=""), patch(
+            "masu.processor.parquet.parquet_report_processor.ParquetReportProcessor.report_type", return_value=None
+        ), patch("masu.processor.parquet.parquet_report_processor.ParquetReportProcessor.prepare_parquet_s3"):
```

Comment:
As of Python 3.10, you may group [with statements](https://docs.python.org/3/reference/compound_stmts.html#the-with-statement) to make them more readable.
```suggestion
        with (
            patch("masu.processor.parquet.parquet_report_processor.get_path_prefix", return_value=""),
            patch("masu.processor.parquet.parquet_report_processor.ParquetReportProcessor.report_type", return_value=None),
            patch("masu.processor.parquet.parquet_report_processor.ParquetReportProcessor.prepare_parquet_s3"),
        ):
```

Reply:
Is this not already grouped?

Reply:
Yes, currently it is a single `with` statement instead of multiple nested `with` statements. But you could group the single statement in `()` in order to break it up over multiple lines in a different way only to enhance readability. They are syntactically the same.